### ISSUE TITLE
Draggable graph instead of sliders

### ISSUE THREAD

**Issue by giangd** [Leeps-Lab/flow_market#90]
# New feature: limit to one buy order and one sell order
_More info in paper notes_
- Sending buy
  - Initially, buy graph is at starting position
  - Handles always opaque, graph always transparent if not in market
  - Click send buy button, blue graph turns opaque, handles disappear
  - Send buy button turns into cancel buy button
  - If click cancel  buy button, then blue graph reverts back to initial condition
  - Implement circles not being able to cross

# Questions

# Description FLO
- Red points start at top
- Blue points start at top
- Can only drag circles vertically
- Can drag squares vertically and horizontally
- Red sq cannot go below red circle
- Blue sq cannot go above blue circle
- Red circle and blue circle cannot cross

![image](https://user-images.githubusercontent.com/13838764/132919405-bddb6d34-7e6f-4795-9583-77b3e6fcee48.png)

# Description CDA
- Red sq start at top
- Blue sq start at bottom
- Red sq and blue sq can move vertically and horizontally
- Red sq cannot cross blue sq
- Basically dragging the corners
![image](https://user-images.githubusercontent.com/13838764/132919575-78081938-c5f0-47f5-93ef-ec65aa301f08.png)

- Algorithmic orders
  - X-axis should still be volume
  - There should be separate input for orders at a time

# Draft
![Untitled](https://user-images.githubusercontent.com/13838764/131931281-bb95e127-ae7d-4dda-a8eb-c922e9a34cbd.png)

# Resources
- https://jsfiddle.net/f72kz348/9/
- https://www.npmjs.com/package/chartjs-plugin-dragdata
